src/components/Staking/StakingStatsBox.js

```diff
@@ -57,12 +57,6 @@ const ErrorMessage = () => (
   </IndicatorSpan>
 )
 
-const LoadingMessage = () => (
```

Comment:
@wackerow we should have a loading state here, ya 😄? Future PR...

Reply:
I removed that because it was not being used.

Reply:
Yep... still needs to be fixed separately